docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -143,6 +143,75 @@ game.properties:        158 bytes
 ui.properties:          83 bytes
 ```
 
+You can pass in the `--from-env-file` to create a ConfigMap from an env-file (i.e. a Docker .env file), for example:
+```shell
+# Env-files contain a list of environment variables.
+# These syntax rules apply:
+#   Each line in an env file has to be in VAR=VAL format.
+#   Lines beginning with # (i.e. comments) are ignored.
+#   Blank lines are ignored.
+#   There is no special handling of quotation marks (i.e. they will be part of the ConfigMap value)).
+
+
+cat docs/tasks/configure-pod-container/game-env-file.properties
+enemies=aliens
+lives=3
+allowed="true"
+
+# This comment and the empty line above it are ignored
+```
+
+```shell
+kubectl create configmap game-config-env-file \
+        --from-env-file=docs/tasks/configure-pod-container/game-env-file.properties
+```
+
+would produce the following ConfigMap:
+
+```shell
+kubectl get configmap game-config-env-file -o yaml
+apiVersion: v1
+data:
+  allowed: '"true"'
+  enemies: aliens
+  lives: "3"
+kind: ConfigMap
+metadata:
+  creationTimestamp: 2017-12-27T18:36:28Z
+  name: game-config-env-file
+  namespace: default
+  resourceVersion: "809965"
+  selfLink: /api/v1/namespaces/default/configmaps/game-config-env-file
+  uid: d9d1ca5b-eb34-11e7-887b-42010a8002b8
+```
+
+Note that when passing `--from-env-file` multiple times to create a ConfigMap from multiple data sources, only the last env-file is used:
```

Comment:
You can remove "Note that" and just start with "When..." unless you're specifically wanting to make a **Note:** callout (https://kubernetes.io/docs/home/contribute/style-guide/#note), in which case you'll want to add `{: .note}` on the line after

Reply:
fixed.